lib/planner/search-algorithms/forward-state-space-search-heuristic.js

```diff
@@ -147,14 +147,22 @@ module.exports = forwardStateSpaceSearch = {
     },
     * _checkPreconditions(node, component, actionName, callback) {
         let next = yield;
+        let action = component.actions[actionName];
 
         if (component.actions[actionName].preconditions === undefined) {
```

Comment:
can just use action here now that you are declaring it

Reply:
Great point. Thanks, I'll fix it now!